src/lib/formatters/html-report/html-report.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @fileoverview The most basic formatter, it just stringifyes whatever object
+ * is passed to it.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+import * as _ from 'lodash';
+import * as fs from 'fs';
+import * as Handlebars from 'handlebars';
+import * as util from 'util';
+import * as path from 'path';
```

Comment:
Global packages come first, and the install ones, and then our own (all in alphabetical order).